tinyagentos/routes/library.py

```
@@ -0,0 +1,626 @@
+    css = _STATUS_CSS.get(status, "status-pending")
+    title = html.escape(item.get("title", "Untitled"))
+    kind = html.escape(item.get("kind", "file"))
+    bytes_val = item.get("bytes") or 0
+    size_str = _format_bytes(bytes_val) if bytes_val else ""
+    item_id = html.escape(item.get("id", ""))
+    source_url = html.escape(item.get("source_url", ""))
+    downloaded = bool(item.get("download_path", ""))
+
+    parts = [
+        f'<div class="item-card" id="item-{item_id}">',
+        f'<div class="info">',
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)